### PR TITLE
worked through and reviewed getting started

### PR DESCRIPTION
**Pull request by StarfallProjects**
Mostly style edits, one or two points where I've moved things around for clarity.

### CODE REVIEW

File: getting-started/index.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ There are three ways to interact with Event Store:
 
 ### [.NET Client](#tab/tabid-dotnet-client)
 
-[Install the .NET Core client API](https://www.nuget.org/packages/EventStore.ClientAPI.NetCore/) using your preferred method, add it to your project:
+[Install the .NET Core client API](https://www.nuget.org/packages/EventStore.ClientAPI.NetCore/) using your preferred method
```
Comment (ChrisChinchilla) [inline getting-started/index.md]:
```suggestion
[Install the .NET Core client API](https://www.nuget.org/packages/EventStore.ClientAPI.NetCore/) using your preferred method.
```

File: getting-started/index.md [hunk 2]
```diff
@@ -81,13 +83,13 @@ When using the .NET client, you also need to give the connection a name.
...
 
-## Writing Events to an Event Stream
+## Writing events to an Event Stream
 
 Event Store operates on a concept of Event Streams, and the first operation we look at is how to write to a stream. If you are Event Sourcing a domain model, a stream equates to an aggregate function. Event Store can handle hundreds of millions of streams, so create as many as you need.
 
 If you post to a stream that doesn't exist, Event Store creates it before adding the events.
 
-You can write events using the Admin UI by clicking the _Stream Browser_ tab, the _Add Event_ button, filling in the form with relevant values and clicking the _Add_ button at the bottom of the page.
+You can write events using the Admin UI by selecting the _Stream Browser_ tab, the _Add Event_ button, filling in the form with relevant values and selecting the _Add_ button at the bottom of the page.
```
Comment (ChrisChinchilla) [inline getting-started/index.md]: Hmm, we may need to talk about this one, in my mind, "selecting" is not the same as "clicking"

File: getting-started/projections.md [hunk 3]
```diff
@@ -6,13 +6,15 @@ outputFileName: index.html
...
+
 When running a projection, you can choose whether the query should run and give you all results present or whether the query should continue running into the future finding new results as they happen and updating its result set.
 
-## Setting up Projections
+## Setting up projections
 
 You enable projections with the command line argument `--run-projections`. For example, the command below enables all projection modes (system and user-defined):
 
-> [!TIP][read this guide](~/server/command-line-arguments.md#projections-options) for all the possible parameter values.
+> [!TIP][Read this guide](~/server/command-line-arguments.md#projections-options) for all the possible parameter values.
```
Comment (ChrisChinchilla) [inline getting-started/projections.md]: This is something that keeps getting broken by my markdown linter, I need to fix it :(

File: getting-started/index.md (hunk 1 above)
Comment (jageall) [reply getting-started/index.md]: The client api netcore package is no longer required, the standard client package includes a .net standard 2.0 target

Comment (jageall) [reply getting-started/index.md]: Ok, that url is actually for entirely the wrong package, the package is actually this one:

**Comment by StarfallProjects** [comment]
> Generally pretty good. What was your thoughts on the headings though? I think I was aiming for sentence case, but I'm not wedded to it. But then you have some headings in title case, and others in sentence case?

I would lean to sentence case also. There was a mix, so I left it and asked you in my questions doc (sent in the email) to confirm: https://docs.google.com/document/d/14F2d52v-z6o278C8jlFcx56UMRgwEoVQ0YCDc3CgpAI/edit?usp=sharing

File: getting-started/index.md (hunk 2 above)
Comment (StarfallProjects) [reply getting-started/index.md]: I generally use "select" when documenting UIs as it covers both "click" and "tap", but to be fair it's unlikely anyone is going to do this on mobile/tablet. Happy to revert to click if you prefer.

File: getting-started/index.md (hunk 1 above)
Comment (StarfallProjects) [reply getting-started/index.md]: @jageall thanks for this, have amended

File: getting-started/index.md (hunk 2 above)
Comment (ChrisChinchilla) [reply getting-started/index.md]: @StarfallProjects OK, noted, but yes, in this case, i don't think anyone is using on mobile, so let's keep as is/was

File: getting-started/index.md (hunk 1 above)
Comment (ChrisChinchilla) [reply getting-started/index.md]: Have you pushed this change @StarfallProjects ?

File: getting-started/index.md (hunk 2 above)
Comment (ChrisChinchilla) [reply getting-started/index.md]: Yeah, let's stick to clicking :)

**Comment by StarfallProjects** [comment]
@ChrisChinchilla any further changes needed here or can we delete?

**Pull request merged**